packages/website/tsconfig.json

```diff
@@ -21,7 +21,7 @@
     },
     "incremental": true
   },
-  "include": ["lib", "pages", "components"],
+  "include": ["lib", "pages", "components", "pages/blog/index/index.js"],
```

Comment:
Does the "pages" declaration not include this?

Reply:
not sure how this got in there